webpack/repositories.bzl

```diff
@@ -27,6 +27,6 @@ def rules_webpack_dependencies():
     maybe(
         http_archive,
         name = "build_bazel_rules_nodejs",
-        sha256 = "cfc289523cf1594598215901154a6c2515e8bf3671fd708264a6f6aefe02bf39",
-        urls = ["https://github.com/bazelbuild/rules_nodejs/releases/download/4.4.6/rules_nodejs-4.4.6.tar.gz"],
+        sha256 = "2b2004784358655f334925e7eadc7ba80f701144363df949b3293e1ae7a2fb7b",
+        urls = ["https://github.com/bazelbuild/rules_nodejs/releases/download/5.4.0/rules_nodejs-5.4.0.tar.gz"],
```

Comment:
nit: do we need to assert somewhere that the user has a minimum version of this?

Reply:
not really. e2e tests still use 4.x. this is here to make sure that rules_webpack still works with never versions of rnjs.